docs/resources/css_cluster.md

```diff
@@ -12,69 +12,122 @@ Manages CSS cluster resource within HuaweiCloud
+  engine_version  = "7.10.2"
+
+  ess_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 1
+    volume {
+      volume_type = "HIGH"
+      size        = 40
+    }
+  }
+
+  availability_zone = var.availability_zone
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
 }
+```
+
+### create a cluster with ess-data node and master node
+
+```hcl
+variable "availability_zone" {}
+variable "vpc_id" {}
+variable "subnet_id" {}
+variable "secgroup_id" {}
 
 resource "huaweicloud_css_cluster" "cluster" {
-  expect_node_num = 1
   name            = "terraform_test_cluster"
-  engine_version  = "7.1.1"
-
-  node_config {
-    flavor = "ess.spec-4u16g"
-    availability_zone = var.availability_zone
+  engine_version  = "7.10.2"
 
-    network_info {
-      security_group_id = huaweicloud_networking_secgroup.secgroup.id
-      subnet_id         =  var.network_id
-      vpc_id            =  var.vpc_id
+  ess_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 1
+    volume {
+      volume_type = "HIGH"
+      size        = 40
     }
+  }
 
+  master_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 3
     volume {
       volume_type = "HIGH"
       size        = 40
     }
   }
+
+  availability_zone = var.availability_zone
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+
 }
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) The region in which to create the cluster resource. If omitted, the
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the cluster resource. If omitted, the
   provider-level region will be used. Changing this creates a new cluster resource.
 
-* `name` - (Required, String, ForceNew) Cluster name. It contains 4 to 32 characters. Only letters, digits, hyphens (-),
-  and underscores (_) are allowed. The value must start with a letter. Changing this parameter will create a new
-  resource.
-
-* `engine_type` - (Optional, String, ForceNew) Engine type. The default value is "elasticsearch". Currently, the value
-  can only be "elasticsearch". Changing this parameter will create a new resource.
+* `name` - (Required, String, ForceNew) Specifies the cluster name. It contains 4 to 32 characters.
+  Only letters, digits, hyphens (-), and underscores (_) are allowed. The value must start with a letter.
+  Changing this parameter will create a new resource.
 
-* `engine_version` - (Required, String, ForceNew) Engine version. Versions 5.5.1, 6.2.3, 6.5.4, 7.1.1, 7.6.2 and 7.9.3
-  are supported. Changing this parameter will create a new resource.
+* `engine_type` - (Optional, String, ForceNew) Specifies the engine type. The default value is `elasticsearch`.
+  The valid values are `elasticsearch`, `logstash`. Changing this parameter will create a new resource.
```

Comment:
logstash is not shown in console, please check it

Reply:
remove it